src/Controller/Adminhtml/Product/CreateProject.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\Connector\Controller\Adminhtml\Product;
+
+use EasyTranslate\Connector\Api\Data\ProjectInterface;
+use EasyTranslate\Connector\Model\Adminhtml\ProjectDataProcessor;
+use EasyTranslate\Connector\Model\Config\Source\Status;
+use EasyTranslate\Connector\Model\Config\Source\Team;
+use EasyTranslate\Connector\Model\ProjectFactory;
+use EasyTranslate\RestApiClient\Workflow;
+use Exception;
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Api\DataObjectHelper;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Ui\Component\MassAction\Filter;
+
+class CreateProject extends Action
+{
+    /**
+     * @var Team
+     */
+    private $team;
+
+    /**
+     * @var ProjectFactory
+     */
+    private $projectFactory;
+
+    /**
+     * @var DataObjectHelper
+     */
+    private $dataObjectHelper;
+
+    /**
+     * @var ProjectDataProcessor
+     */
+    private $projectDataProcessor;
+
+    public function __construct(
+        Context $context,
+        Team $team,
+        DataObjectHelper $dataObjectHelper,
+        ProjectDataProcessor $projectDataProcessor,
+        ProjectFactory $projectFactory
+    ) {
+        parent::__construct($context);
+        $this->team                 = $team;
+        $this->projectFactory       = $projectFactory;
+        $this->dataObjectHelper     = $dataObjectHelper;
+        $this->projectDataProcessor = $projectDataProcessor;
+    }
+
+    public function execute()
+    {
+        $resultRedirect = $this->resultRedirectFactory->create();
+        try {
+            $data = $this->projectData();
+        } catch (Exception $exception) {
+            $this->messageManager->addErrorMessage(__('Could not create project. Please check your credentials'));
+
+            return $resultRedirect->setPath('catalog/product/edit/id/' . $this->getProductIds()[0]);
+        }
+        $project = $this->projectFactory->create();
+        $this->dataObjectHelper->populateWithArray($project, $data, ProjectInterface::class);
+        $project = $this->projectDataProcessor->saveProjectPostData($project, $data);
+        $this->messageManager->addSuccessMessage(
+            (string)__('The project has successfully been created.
+        Please change the settings according to your needs before you send the project to EasyTranslate.')
```

Comment:
Are you sure multi-line strings are supported by the translation collection console command? By the way, please run it to update the i18n file.